indexer/feature_data.cpp

```diff
@@ -149,6 +149,7 @@ class UselessTypesChecker
         {"amenity", "shelter"},
         {"amenity", "toilets"},
         {"amenity", "drinking_water"},
+        {"shop", "copyshop"}, // often used as secondary tag for amenity=post_office
```

Comment:
@vng there is a clear "secondary" signal from the original OSM tags (e.g. amenity= is primary, *=yes is secondary), why we can't use it automatically?